### PR TITLE
Add New Updated Flag Sample Page

### PR DESCRIPTION
**Pull request by hyork**
Add a sample page to show how to use DATE_DIFF function and template field to show a new/updated flag to records on a table

### CODE REVIEW

**Review by Rnhatch — COMMENTED** [review]
This looks really interesting.  Can you also update the readme in the "Skuid Techniques"  directory itself with a Link to this item?   That page serves as a sort of TOC.

File: Skuid_Techniques/New_Updated_Flag/README.MD [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Add a flag for new or recently updated records on table
```
Comment (itmightbecody) [inline Skuid_Techniques/New_Updated_Flag/README.MD:1]:
```suggestion
# Add a flag for new or recently updated records on a Table component
```

File: Skuid_Techniques/New_Updated_Flag/README.MD [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+<img src="new-updated-flag.png" width="400"></img>
+
+## Instructions
+- Page API:  V2
+- Data source: Uses default Salesforce data source
+- Design system: ProductTour (available out of the box)
+- Page XML:  [Copy the XML from this page](NewUpdatedFlag.xml), or save it as an XML file, and upload it as a new page in your Salesforce Org.
```
Comment (itmightbecody) [inline Skuid_Techniques/New_Updated_Flag/README.MD:9]:
```suggestion
- Page XML:  [Copy the XML from this page](NewUpdatedFlag.xml?raw=true), or save it as an XML file, and upload it as a new page in your Salesforce Org.  
```

File: Skuid_Techniques/New_Updated_Flag/README.MD (hunk 1 above)
Comment (itmightbecody) [reply Skuid_Techniques/New_Updated_Flag/README.MD:1]: Style guide stuffz

File: Skuid_Techniques/New_Updated_Flag/README.MD (hunk 2 above)
Comment (itmightbecody) [reply Skuid_Techniques/New_Updated_Flag/README.MD:9]: This lets users download the file super quick!

File: Skuid_Techniques/New_Updated_Flag/README.MD [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<img src="new-updated-flag.png" width="400"></img>
+
+## Instructions
+- Page API:  V2
+- Data source: Uses default Salesforce data source
+- Design system: ProductTour (available out of the box)
+- Page XML:  [Copy the XML from this page](NewUpdatedFlag.xml?raw=true), or save it as an XML file, and upload it as a new page in your Salesforce Org.  
+
+## Notes
+This sample page highlights DATE_DIFF function and template field on table component.
```
Comment (itmightbecody) [inline Skuid_Techniques/New_Updated_Flag/README.MD:12]:
```suggestion
This sample page highlights DATE_DIFF function and template field on a Table component.
```

File: Skuid_Techniques/New_Updated_Flag/README.MD [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- To see the Updated flag in action, make a change to a record and click Save
+- To see the New flag in action, create a new record and Save
+- The formula is currently set to show the flag if the record is new or updated within an hour. To change the unit (to a day for example), update the unit in the DATE_DIFF function from HOUR to DAY
+
+
+## Related Links
+- [Date diff function](https://docs.skuid.com/latest/en/skuid/formula-function.html#date-time-format-and-manipulation)
```
Comment (itmightbecody) [inline Skuid_Techniques/New_Updated_Flag/README.MD:19]:
```suggestion
- [DATE_DIFF function documentation](https://docs.skuid.com/latest/en/skuid/formula-function.html#date-time-format-and-manipulation)
```

**Review by itmightbecody — CHANGES_REQUESTED** [review]
Ack, I should've bundled those other style suggestions with as an actual PR review.

This is a cool example with good notes. Could you add these final style suggestions? Then this is good to go!

**Comment by hyork** [comment]
Thanks Cody!

**Review by itmightbecody — APPROVED** [review]
Looks great! Thank you Huyen! 😁

**Pull request merged**